VARIABLES.md

```diff
@@ -9,7 +9,7 @@
 | apiary_domain_name | Apiary domain name for Route 53. | string | `` | no |
 | apiary_log_bucket | Bucket for Apiary logs. | string | - | yes |
 | apiary_log_prefix | Prefix for Apiary logs. | string | `` | no |
-| apiary_managed_schemas | Schema names from which S3 bucket names will be derived, corresponding S3 bucket will be named as apiary_instance-aws_account-aws_region-schema_name. | list | `<list>` | no |
+| apiary_managed_schemas | Schema names from which S3 bucket names will be derived, corresponding S3 bucket will be named as apiary_instance-aws_account-aws_region-schema_name, along with s3 storage properties like storage class and no of days for transitions. | list of map | `<list of map>` | no |
```

Comment:
I see the type has changed here. What are the keys and values in the map? Also, is this a backwards incompatible change for end users? If so the version number should go to 2.0.0 and we should clearly mention this change in the CHANGELOG so users migrating know what they would need to do when they upgrade.

Reply:
Hey where is a good place to add the valid keys and values in the map? This change is backward incompatible